src/components/groups/GroupContainer.js

```diff
@@ -30,28 +31,30 @@ function GroupContainer() {
 				</>
 			</MyGroups>
 			<Divider />
+      <NearbyGroups />
+      <Divider />
 			<GroupList />
 		</div>
 	);
 }
 export default GroupContainer;
 
 const MyGroups = styled.div`
-	margin: 2% 5%;
+  margin: 2% 5%;
 `;
 
 const GroupTitleHolder = styled.div`
-	display: flex;
-	flex-direction: row;
-	justify-content: flex-start;
-	align-items: center;
-	h3 {
-		margin: 0 1%;
-	}
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  h3 {
+    margin: 0 1%;
+  }
 `;
 
 const H3 = styled.h3`
-	font-size: 1rem;
-	font-weight: bold;
-	margin: 1% 0;
+  font-size: 1rem;
+  font-weight: bold;
+  margin: 1% 0;
```

Comment:
Looks like we have some prettier inconsistencies among the group -- we should collaborate and make sure everybody has the same extension and settings